QtCollider/widgets/QcWebView.h

```diff
@@ -116,8 +116,12 @@ public Q_SLOTS:
     Q_PROPERTY(QSizeF contentsSize READ contentsSize)
     QSizeF contentsSize() const { return page() ? page()->contentsSize() : QSizeF(0, 0); }
 
-    Q_PROPERTY(QPointF scrollPosition READ scrollPosition)
+    Q_PROPERTY(QPointF scrollPosition READ scrollPosition WRITE setScrollPosition)
     QPointF scrollPosition() const { return page() ? page()->scrollPosition() : QPointF(0, 0); }
+    void setScrollPosition(QPointF p) {
+        if (page())
+            page()->runJavaScript(QString("window.scrollTo(%1, %2);").arg(p.rx()).arg(p.ry()));
```

Comment:
is this really the correct way to do this? running JS to set a scroll position feels... ok? strange? Curious if there isn't another way to do this more 'natively'

Reply:
I was wondering about the same and found on the qt forum that indeed suggesting to use JS, as webengine view doesn't seem to provide such functionality atm.

Reply:
yes, it is strange I agree but @dyfer is correct.